src/Build/Definition/Project.cs

```diff
@@ -109,6 +109,7 @@ internal Project(ProjectCollection projectCollection, ProjectLink link)
             ProjectCollection = projectCollection;
             implementationInternal = new ProjectLinkInternalNotImplemented();
             implementation = link;
+            StringToolsHost.Initialize();
```

Comment:
This could be called in a static Project constructor right? To make it more clear that it's a static, once in a process thing.

Reply:
Indeed, I've moved it to the static ctor, thanks!